tests_e2e/orchestrator/docker/Dockerfile

```diff
@@ -71,7 +71,7 @@ RUN \
     cd $HOME                                                                                                  && \
     git clone https://github.com/microsoft/lisa.git                                                           && \
     cd lisa                                                                                                   && \
-    git checkout 0e37ed07304b74362cfb3d3c55ac932d3bdc660c                                                     && \
+    git checkout 99cfc195f9285fb0188ef6f24a9f613dc203e897                                                     && \
```

Comment:
They fix the lisa template to support defaultoutboundaccess property https://github.com/microsoft/lisa/pull/3705